helm-projectile.el

```diff
@@ -237,7 +237,12 @@ It is there because Helm requires it."
   "Helm source for known projectile projects.")
 
 (cl-defmethod helm--setup-source :after ((source helm-projectile-projects-source))
-  (setf (slot-value source 'action) 'helm-source-projectile-projects-actions))
+  (setf (slot-value source 'action) 'helm-source-projectile-projects-actions)
+  (setf (slot-value source 'keymap) helm-projectile-projects-map)
+  (setf (slot-value source 'persistent-action) nil)
```

Comment:
I'm not sure what `persistent-action` is, so perhaps add a note about it and why we need to disable it.

Reply:
Originally I just set it up to `helm-source` defaults, since `helm-projectile-projects-source` doesn't define them. But this comment made me to revisit it and I think it's better to literally do nothing for now. Until some better idea of how to "peek into a project" may look like.